### PR TITLE
Fill in missing introduction module

### PR DESCRIPTION
**Pull request by adamlazik1**
While working on #2414, I did not realize that I forgot to write introduction to Using Repository Sources subchapter. The sections then went under *Configuring the Templates Plugin* which was mildly related so the procedures did not stick out as misplaced. After reviewing PR again I noticed an empty introduction module.

I also rephrased introduction in a subsequent module to not repeat the same thing twice in the docs.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (planned Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (Satellite 6.13)
* [x] Foreman 3.4/Katello 4.6 (EL8 only)
* [x] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* [x] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [x] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* We do not accept PRs for Foreman older than 3.1.

### CODE REVIEW

File: guides/common/modules/con_using-repository-sources.adoc [hunk 1]
```diff
@@ -1 +1,5 @@
 [id="using-repository-sources_{context}"]
+
```
Comment (maximiliankolb) [inline guides/common/modules/con_using-repository-sources.adoc:2]:
```suggestion
```

File: guides/common/modules/proc_synchronizing-templates-with-an-existing-repository.adoc [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [id="synchronizing-templates-with-an-existing-repository_{context}"]
 = Synchronizing Templates with an Existing Repository
 
-You can use existing repositories to import and export templates from your {ProjectServer}.
+Use this procedure to synchronize templates between your {ProjectServer} and an existing repository.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_synchronizing-templates-with-an-existing-repository.adoc:4]: Can you/are you meant to sync both ways? If it's one way from source > Foreman, then maybe reword from "between" to "to". I am unsure how this is meant to be used; it's more of a question.

File: guides/common/modules/con_using-repository-sources.adoc (hunk 1 above)
Comment (adamlazik1) [reply guides/common/modules/con_using-repository-sources.adoc]: Applied.

File: guides/common/modules/proc_synchronizing-templates-with-an-existing-repository.adoc (hunk 2 above)
Comment (adamlazik1) [reply guides/common/modules/proc_synchronizing-templates-with-an-existing-repository.adoc]: See if the new wording makes more sense.

Comment (maximiliankolb) [reply guides/common/modules/proc_synchronizing-templates-with-an-existing-repository.adoc]: If it's both ways, then your first wording was totally OK for me.

Comment (adamlazik1) [reply guides/common/modules/proc_synchronizing-templates-with-an-existing-repository.adoc:4]: reworded back.

**Review by maximiliankolb — APPROVED** [review]

**Comment by adamlazik1** [comment]
Merged and cherry-picked:
   c8b010d0a..77a7dd838  3.1 -> 3.1
   398a69c45..dcd237d94  3.2 -> 3.2
   80e48fa3b..91ee05b91  3.3 -> 3.3
   ea1587989..73ab984a9  3.4 -> 3.4
   429188277..0ff301ee6  3.5 -> 3.5
   08dc0ff28..de791a58b  3.6 -> 3.6
   848362782..a4adeeabd  3.7 -> 3.7
   d48db6cff..c6ffbe888  3.8 -> 3.8
